tasks/utils/emission-disperse-bal.ts

```diff
@@ -0,0 +1,115 @@
+import https from "https"
+import { BN, simpleToExactAmount } from "@utils/math"
+
+const MTA_ADDRESS = "0xF501dd45a1198C2E1b5aEF5314A68B9006D842E0"
```

Comment:
You can get the address from `PMTA.address`

Reply:
resolved with e270e136bd3e196c30033659347192d30ff84da0